src/SFA.DAS.EmployerAccounts.Web/Orchestrators/HomeOrchestrator.cs

```diff
@@ -71,5 +71,18 @@ public virtual async Task SaveUpdatedIdentityAttributes(string userRef, string e
                 FirstName = firstName
             });
         }
+
+        public virtual async Task<OrchestratorResponse<UserAccountsViewModel>> GetAccounts()
```

Comment:
Can we keep this method name consistent with the GetUserAccounts(string userId) method name?

Reply:
removed